app.rb

```diff
@@ -0,0 +1,84 @@
+require 'sinatra'
+require 'sinatra/reloader'
+require 'http'
+require 'rack-livereload'
+require 'lazyrecord'
+require_relative 'model/Book'
+
+use Rack::LiveReload if development?
+
+@@key = "AIzaSyC8xY7H9mf9HVrAmFPcb66v9hWeW4asmBU"
+helpers do
+  def get_books(params)
+    JSON.parse(HTTP.get("https://www.googleapis.com/books/v1/volumes?q=#{params}&key=#{@@key}").body)
```

Comment:
There is no need to add build the query params yourself here `?q=#{params}&key=#{@@key}`, instead,
use the `params` keyword from `http`.
https://github.com/httprb/http/wiki/Passing-Parameters#query-string-parameters

```suggestion
    JSON.parse(HTTP.get("https://www.googleapis.com/books/v1/volumes", params: { q: params, key: @@key }).body)
```